websites/Z/Zybooks/presence.ts

```diff
@@ -0,0 +1,104 @@
+/* eslint-disable one-var */
+/* eslint-disable no-one-time-vars/no-one-time-vars */
```

Comment:
why

Reply:
I think one-time variables make code more readable, as it is somewhat self-documenting.

Reply:
While you and others may think that these rules make it less readable, this repository enforces these rules to be create consistent code and what the repository owners may consider as more readable. For the time being, please remove the disabling of the eslint rules. Also, thanks for your contribution!
```suggestion
```